autoware_map_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+project(autoware_map_msgs)
+
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+set(msg_files
+  "msg/AreaInfo.msg"
+  "msg/PointCloudMapWithID.msg"
+  "srv/PartialPointCloudMap.srv"
+  "srv/DifferentialPointCloudMap.srv")
```

Comment:
I think `srv` files generally start from a verb.

Reply:
Right, thanks!

1. `LoadDifferentialPointCloudMap.srv` (<- a bit too long)
2. `LoadDifferentialPCDMap.srv` (<- shorter, but depends on whether "pcd" is familiar enough for users)
3. `LoadPCDMapDifferentially.srv` (<- sounds more natural to me)

May I ask your thoughts on this? @kenji-miyake @yukkysaito @mitsudome-r

Reply:
Mm, I don't understand what this service does exactly, but I personally prefer 1. I'll leave it to you.

Reply:
Resolved https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/21fa676ce73df4fa56519665e58d00c967aec431